rust/Cargo.toml

```diff
@@ -97,6 +99,7 @@ members = [
 [dev-dependencies]
 tracing-test = "0.2.5"
 criterion = "0"
+once_cell = "1.20.1"
```

Comment:
This is needed to lock the test server behind a static mutex. I think we very likely indirectly depend on this anyways, since it is a pretty common library.